### DISCUSSION TITLE
EasyToll (13/01/2023)

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#42] [Esami]
In questa discussion a tema il progetto "EasyToll", oggetto del testo di esame del 13 gennaio 2023, discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by progettistamatto** [comment]
Per questo esame è disponibile [[QUI]](https://github.com/Exyss/university-notes/raw/main/Bachelor/Secondo%20Anno/Basi%20di%20Dati%20II/EasyToll.pdf) una soluzione proposta da Simone Bianco (Exyss)

> [!Warning]
_Le soluzioni di Simone Bianco fanno uso degli **Entity-Relationship (ER)** Diagrams su cui il corso si basava fino all'Anno Accademico 2023/2024, mentre **attualmente vengono utilizzati gli UML Class** Diagram. Sebbene i due siano estremamente simili, vi sono **alcune discordanze tra di loro**, in particolare l'**inversione di posizione dei vincoli di molteplicità agli estremi delle relationship**. Nonostante ciò, il rimanente delle fasi di analisi e progettazione delle soluzioni proposte risulta inaffetto da tale cambiamento. Per tanto, una volta comprese le differenze tra ER e Classi UML, le soluzioni risultano normalmente fruibili._

Rispondi a questo commento taggando @Exyss per eventuali dubbi o segnalazioni in merito

**Reply by Loriv3** [reply]
Ciao, nei vincoli spesso leggo V.<classe>.Disgiunti non riesco a capire bene sia il perchè sia cosa significhi il vincolo

**Comment by progettistamatto** [comment]
Per questo progetto è disponibile [[QUI]](https://github.com/SimoneLid/BD2/raw/main/Prove%20D'Esame/2023-1-13-EasyToll/EasyToll.pdf) una soluzione proposta da Simone Lidonnici (@SimoneLid)
